test/Locker.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Contract } from "ethers";
+import { expect } from "./chai-setup";
+import { pack, keccak256 } from "@ethersproject/solidity";
+import { ethers } from "hardhat";
+import {
+  setupNftGemGovernor
+} from './fixtures/Governance.fixture';
+
+describe('Locker Test Suite', function () {
+  let locker: Contract;
+  let NFTGemMultiToken: any;
+  const tokenHash = keccak256(['bytes'], [pack(['string'], ['Test Token'])]);
+  beforeEach(async () => {
+    // Deploy Locker Contract on each test iteration
+    locker = await (await ethers.getContractFactory('Locker')).deploy();
+    const setupNftGemGovernorResult = await setupNftGemGovernor();
+    NFTGemMultiToken = setupNftGemGovernorResult.NFTGemMultiToken;
+    const [sender] = await ethers.getSigners();
+    // Mint token to sender to make sure the sender have enough balance
+    await NFTGemMultiToken.mint(sender.address, tokenHash, 100);
+    await NFTGemMultiToken.connect(sender).setApprovalForAll(locker.address, true);
+  })
+  it('Drop Off', async function () {
+    await locker.dropOff(
+      NFTGemMultiToken.address,
+      tokenHash,
+      NFTGemMultiToken.address,
+      tokenHash,
+      5);
+    let lockerContent = await locker.contents(tokenHash);
```

Comment:
This test has no comments. There are several logical operations here but zero comments as to what is going on. That's not acceptable. Look at it this way: At some point, future Firrly, or someone else, is going to come reading this file. You have the power right now to make that future person's day a little happier by leaving them a piece of code that takes them minimal energy to easily understamd. Or, you can annoy someone in the future by leaving a mess. There is no saving time by omitting comments. Someone elways has to pay and usually it's you.